src/002_installation.md

```diff
@@ -55,48 +55,49 @@ Continue as described in Section [Running Tamarin](#sec:running-tamarin) to run
 Mac OS X {#sec:macosx}
 --------
 
-### Automatic Installation of the Tamarin binary {#sec:MacOSBinInstall}
+### Automatic Installation of Tamarin {#sec:MacOSBinInstall}
 
-The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/) [Cask](https://caskroom.github.io/).
+The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/).
 
 If you already have this installed, it is as simple as running the following two commands in your terminal:
 
   * `brew update`
-  * `brew cask install tamarin-prover`
+  * `brew install homebrew/science/tamarin-prover`
```

Comment:
`homebrew/science` isn't needed I think. Since science is in the brew core. 
Alternatively can add a point above asking them to Tap `brew tap homebrew/science`